parser/parser.go

```diff
@@ -72,6 +83,24 @@ func Parse(reader io.Reader) ([]Command, error) {
 	return commands, scanner.Err()
 }
 
+func Format(w io.Writer, commands []Command) error {
```

Comment:
Would it simpler/less code to just rewrite the `FROM` or `ADAPTER` arguments vs the entire Modelfile?

Reply:
Updated to do a `bytes.Replace` instead